attribute_set/models/attribute_set_owner.py

```diff
@@ -33,18 +33,22 @@ def _build_attribute_eview(self):
         if not self._context.get("include_native_attribute"):
             domain.append(("nature", "=", "custom"))
 
-        attributes = self.env["attribute.attribute"].search(domain)
+        attributes = self.env["attribute.attribute"].sudo().search(domain)
         return attributes._build_attribute_eview()
 
     @api.model
     def remove_native_fields(self, eview):
         """Remove native fields related to native attributes from eview"""
-        native_attrs = self.env["attribute.attribute"].search(
-            [
-                ("model_id.model", "=", self._name),
-                ("attribute_set_ids", "!=", False),
-                ("nature", "=", "native"),
-            ]
+        native_attrs = (
```

Comment:
why the parenthesis?
why no simply add sudo()?

Reply:
Hi, I got this suggestion in the pre-commit check: https://github.com/brain-tec/odoo-pim/runs/5315524570?check_suite_focus=true

Reply:
thanks for the clarification :)